internal/cmd/beta/beta_cmd.go

```diff
@@ -10,8 +13,10 @@ import (
 )
 
 type commonFlags struct {
+	silent     bool
```

Comment:
Nit: I started organizing fields alphabetically when there is no apparent ordering dictated by business logic.

Reply:
:+1: